SCF-Session-5/lcmhcf/LcmHcfTest.java

```diff
@@ -0,0 +1,36 @@
+package lcmhcf;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class LcmHcfTest 
+{
+	LcmHcf lcmHcfObject = new LcmHcf();
+	
+	@Test
+	public void lcmTest() 
+	{
+		int lcmOutput = lcmHcfObject.lcm(24, 60);
+		assertEquals(120, lcmOutput);
+		
+		int hcfOutput = lcmHcfObject.hcf(24, 60);
+		assertEquals(12, hcfOutput);
+
+	}
+	
+	@org.junit.Test(expected = AssertionError.class)
```

Comment:
Why it is `@org.junit.Test` while others are just `@Test`?

Reply:
done